docs/media-release/index.html

```diff
@@ -0,0 +1,198 @@
+		</script>
+	<![endif]-->
+
+</head>
+
+<!-- BODY -->
+<!-- GRID-12 -->
+<body class="au-grid">
+
+	<!-- SKIP-LINKS -->
+	<nav class="au-skip-link">
+		<a class="au-skip-link__link" href="#content">Skip to main content</a>
+		<a class="au-skip-link__link" href="#mainmenu">Skip to main navigation</a>
+	</nav>
+
+	<!-- HEADER -->
+	<header class="au-header au-header--dark" role="banner">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-12">
+					<a class="au-header__brand" href="#">
+						<img class="au-header__brand-image" alt="Insert alternate text here" src="../assets/img/header-logo-agov.png">
+						<div class="au-header__text">
+						<h2 class="au-header__heading">Name of website</h2>
+						<div class="au-header__subline">Simple explaination of the website</div>
+						</div>
+					</a>
+				</div>
+			</div>
+		</div>
+	</header>
+
+	<!-- MAIN NAVIGATION -->
+	<nav class="au-main-nav au-main-nav--dark" id="mainmenu" aria-label="main">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-12">
+					<div class="au-main-nav__content" id="main-nav-default-dark">
+						<button
+							aria-controls="main-nav-default-dark"
+							class="au-main-nav__toggle au-main-nav__toggle--open"
+							onClick="return AU.mainNav.Toggle( this )">
+							Menu
+						</button>
+						<div class="au-main-nav__menu">
+							<div class="au-main-nav__menu-inner">
+								<div class="au-main-nav__focus-trap-top"></div>
+								<button
+									aria-controls="main-nav-default-dark"
+									class="au-main-nav__toggle au-main-nav__toggle--close"
+									onClick="return AU.mainNav.Toggle( this )">
+									Close
+								</button>
+								<ul class="au-link-list">
+									<li><a href="../">Home</a></li>
+									<li><a href="../basic">Basic page</a></li>
+									<li><a href="../form/">Form page</a></li>
+									<li class="active"><a href="#">Media release</a></li>
+									<li><a href="#">Distinct from eachother</a></li>
+								</ul>
+								<div class="au-main-nav__focus-trap-bottom"></div>
+							</div>
+						</div>
+						<div class="au-main-nav__overlay" aria-controls="main-nav-default-dark" onClick="return AU.mainNav.Toggle( this )"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</nav>
+
+	<!-- MAIN BODY -->
+	<main id="content">
+		<!--CONTENT-->
+		<div class="au-body container-fluid">
+			<div class="row">
+			<article class="col-md-7 media-release">
+				<h1>Third release of the Trusted Digital Identity Framework</h1>
+				<p class="media-release__caption">
```

Comment:
Use of the `au-display-*` classes seems more natural here, but brings additional visual weight. 

Could we explore a `au-paragraph-introduction` option to keep it generic?

Reply:
`au-display--` doesn't apply the appropriate color shading. I've used the AU-fontgrid() mixin instead to adjust the size.

This is using [BEM](http://getbem.com/naming/)

We can rename it to `media-release__introduction`